services/src/main/java/org/exoplatform/addons/gamification/rest/RealizationsRest.java

```diff
@@ -110,8 +118,11 @@ public Response getAllRealizations(@Parameter(description = "result fromDate", r
 
     try {
       List<GamificationActionsHistoryDTO> gActionsHistoryList =
-          realizationsService.getAllRealizationsByFilter(filter, offset, limit);
+          realizationsService.getRealizationsByFilter(filter, identity, offset, limit);
       return Response.ok(GamificationActionsHistoryMapper.toRestEntities(gActionsHistoryList)).build();
+    } catch (IllegalAccessException e) {
+      LOG.debug("A not authorized user '{}' isn't allowed to access realizations. (earnerId = {})", currentUser, earnerId, e);
+      return Response.status(Response.Status.FORBIDDEN).entity("user is not autorized").build();
```

Comment:
BTW, authorized instead of ~~autorized~~

```suggestion
      return Response.status(Response.Status.FORBIDDEN).build();
```